src/screens/WaitTimeScreen.tsx

```diff
@@ -320,7 +321,7 @@ const styles = StyleSheet.create({
     paddingBottom: 0,
   },
   searchContainer: {
-    paddingTop: 35,
+    paddingTop: Platform.OS === "ios" ? 35 : 0,
```

Comment:
Gonna test this, waiting for build to finish ugh

Reply:
Fucking clipboard 😡